project/make.sh

```diff
@@ -99,14 +99,10 @@ if [ "$DOCKER_EXECDRIVER" = 'lxc' ]; then
 fi
 
 # Use these flags when compiling the tests and final binary
-LDFLAGS='
-	-X '$DOCKER_PKG'/dockerversion.GITCOMMIT "'$GITCOMMIT'"
-	-X '$DOCKER_PKG'/dockerversion.VERSION "'$VERSION'"
-'
 
-if [ -z "$DEBUG" ]; then
-	LDFLAGS="-w $LDFLAGS"
-fi
+source hack/make/.dockerversion
+IAMSTATIC="true"
```

Comment:
Shouldn't this be set before we `source hack/make/.dockerversion`?

Could we please instead do `source "$(dirname "$BASH_SOURCE")/make/.dockerversion"` here too so it's consistent with the other usages and encodes less assumptions?

As a final thought on this line, the double quotes (`"`) here and on the next line (`LDFLAGS`) could/should just be single quotes (`'`).